wordle-web/src/views/WordleView.vue

```diff
@@ -89,5 +129,15 @@ function keyPress(event: KeyboardEvent) {
 
 function onOverlay(name: string) {
   input.value = !input.value
+  playerName.value = name
+  Axios.post('player', {
```

Comment:
![image](https://github.com/schemcharger/EWU-CSCD379-2023-Spring/assets/43832670/341f358d-4cd9-406a-9028-5a7338c3f591)

Looks like this endpoint doesn't exist? I think you called the endpoint 'UpdatePlayer'